### PR TITLE
Fix unwanted skip

### PR DESCRIPTION
**Pull request by deltakosh**

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/15664/merge

Test environment: 
https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15664/merge/index.html

To test a playground add it to the URL, for example:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15664/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/15664/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/15664/merge
https://gui.babylonjs.com/?snapshot=refs/pull/15664/merge
https://nme.babylonjs.com/?snapshot=refs/pull/15664/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/15664/merge#BCU1XR#0

**Comment by bjsplat** [comment]
Visualization tests for WebGPU (Experimental)

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15664/merge/testResults/webgpuplaywright/index.html

**Comment by bjsplat** [comment]
WebGL2 visualization test reporter:

https://snapshots-cvgtc2eugrd3cgfd.z01.azurefd.net/refs/pull/15664/merge/testResults/webgl2playwright/index.html

File: packages/dev/core/src/scene.ts [hunk 1]
```diff
@@ -304,7 +304,13 @@ export class Scene implements IAnimatable, IClipPlanesHolder, IAssetContainer {
         this.onScenePerformancePriorityChangedObservable.notifyObservers(value);
     }
 
+    /** @internal */
+    public get _pickingAvailable() {
```
Comment (sebavan) [inline packages/dev/core/src/scene.ts:308]: Why not using a static and simply change the value instead of overriding a prop ?

Comment (deltakosh) [reply packages/dev/core/src/scene.ts:308]: This is a great point. I reintroduced what was there but static is better!

**Comment by ryantrem** [comment]
There is still a commented out section with the old `// Object.defineProperty(Scene.prototype, "_pickingAvailable", {`, will you remove that in this PR as well?

**Comment by deltakosh** [comment]
Gosh I'm old..

**Review by ryantrem — APPROVED** [review]

**Review by sebavan — APPROVED** [review]

**Pull request merged**